core/server/src/main/java/alluxio/master/file/FileSystemMasterClientRestServiceHandler.java

```diff
@@ -276,8 +278,8 @@ public Response loadMetadata(@QueryParam("path") String path,
       @QueryParam("recursive") boolean recursive) {
```

Comment:
should be removed completely ... REST API is new in 1.1 and we do not need to add deprecated endpoints